addons/web/static/src/core/autocomplete/autocomplete.js

```diff
@@ -393,12 +395,14 @@ export class AutoComplete extends Component {
                 if (!this.isOpened) {
                     this.open(true);
                 }
+                this.scroll();
```

Comment:
What if we're opening the dropdown? Does it scroll?
It's missing tests.

Reply:
- If you're opening the dropdown, `this.activeSourceOptionId` isn't set, so `this.scroll()` returns and does nothing.
- The test "Rounding, value suggestions, keyboard only" in `addons/analytic/static/tests/analytic_distribution_tests.js` actually already tests the arrow behavior and highlighted that I needed to check `isScrollable()`.

I added a proper test to check that the option is fully within view, but I had to create a new hoot dom function for this, as the existing ones don't check the scrollable element (`isVisible` or `isInViewPort`).

Reply:
@mcm-odoo did you have a chance to look at this? :)